C/other/Permutation_of_string.c

```diff
@@ -0,0 +1,65 @@
+// program to generate all permutation of the given string
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+//declare permutation and swap functions
+void permutation(char *,int,int);
+void swap(char *,char *);
+int main()
+{
+    char *s;
+    // dynamically creating string length
+    s=(char*)malloc(sizeof(char)*1000);
+    // getting string
+    gets(s);
+    // changing size to the length of string+1 to store null at end
+    s=realloc(s,strlen(s)+1);
+	//calling permutation
+    permutation(s,0,strlen(s)-1);
```

Comment:
Display a message in print form as what are you showing output to the user for better UI

Reply:
I didn't understand what to show can you tell me

Reply:
You need to print message like this 
`The Permutation of the string are:`
Hope it makes sense. Do, same for input also.

Reply:
ok thanks